include/cantera/thermo/PDSS.h

```diff
@@ -158,10 +158,10 @@ class PDSS
     PDSS& operator=(const PDSS& b) = delete;
     virtual ~PDSS() {}
 
-     //! @}
-     //! @name Molar Thermodynamic Properties of the Species Standard State in
-     //!     the Solution
-     //! @{
+    //! @}
+    //! @name Molar Thermodynamic Properties of the Species Standard State in
+    //!     the Solution
```

Comment:
This gets formatted with `the Solution` being interpreted as the extended description rather than part of the heading. In this case, I think the easiest thing to do is just drop the excessively verbose "in the Solution" suffix.

Reply:
I shortened this (and corresponding instances)